SparkPost/matchbox
# UX-409 & UX-423 Update Panel padding to 450, Update Expandable styling

### What Changed
- Updates padding to use `450`
- UX-423 - Also updates borderless expandable styling
-
### How To Test or Verify
- Run storybook and verify padding on Tables, Panels work as expected
- Visit http://localhost:9001/?path=/story/layout-expandable--borderless and verify borderless expandables work as expected with the new styles

### PR Checklist
